apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -299,6 +288,7 @@ export const getLocalisedSitemap = async (
               "Page",
               "CollectionPage",
               "RootPage",
+              "Collection",
```

Comment:
nit: should use `ResourceType`.

Reply:
not doing - these have the types automatically inferred so very little value add. ie, we can't do `Collections` (for eg) here, because it's a type error